websites/management/commands/backpopulate_groups.py

```diff
@@ -0,0 +1,36 @@
+""" Backpopulate website groups and permissions"""
+from django.core.management import BaseCommand
+
+from main.utils import now_in_utc
+from websites.models import Website
+from websites.permissions import create_global_groups, create_website_groups
+
+
+class Command(BaseCommand):
```

Comment:
Related to my comment below: there can be a potentially huge amount of sites in the db. We should add optional parameters to this command to target specific websites based on `name` or `title`.

Reply:
Okay, though the intended purpose of this command is to create permission groups for all existing websites, or conceivably to add new groups/permissions to all websites if we add any in the future.  I was originally going to do it as a migration script but didn't want a migration to run that long so chose to go this route.